From b2ee78de50a26b0ddc8cfd0cf637d0a3649508b9 Mon Sep 17 00:00:00 2001
From: Miklos Szeredi <mszeredi@suse.cz>
Date: Thu, 30 Aug 2012 16:13:49 +0200
Subject: [PATCH 02/13] vfs: export do_splice_direct() to modules
Patch-mainline: not yet

Export do_splice_direct() to modules.  Needed by overlay filesystem.

Signed-off-by: Miklos Szeredi <mszeredi@suse.cz>
---
 fs/splice.c |    1 +
 1 file changed, 1 insertion(+)

Index: linux-3.6-rc7-master/fs/splice.c
===================================================================
--- linux-3.6-rc7-master.orig/fs/splice.c	2012-09-24 03:10:57.000000000 +0200
+++ linux-3.6-rc7-master/fs/splice.c	2012-09-28 13:36:49.000000000 +0200
@@ -1308,6 +1308,7 @@ long do_splice_direct(struct file *in, l
 
 	return ret;
 }
+EXPORT_SYMBOL(do_splice_direct);
 
 static int splice_pipe_to_pipe(struct pipe_inode_info *ipipe,
 			       struct pipe_inode_info *opipe,
